docs/man/docker-images.1.md

```diff
@@ -34,7 +34,9 @@ versions.
    Show all images (by default filter out the intermediate image layers). The default is *false*.
 
 **-f**, **--filter**=[]
-   Provide filter values (i.e., 'dangling=true')
+   Provide filter values. Valid filters:
```

Comment:
Yeah, I wasn't sure whether you were showing a new kind of filter or interpreting dandling=true...thanks for the explanation.  It sounds like you are saying this is a cut and paste error --- that you cannot use the dandling filter with the images.  So

-->

Filter by label, for example,`label=example.com=amd64` or `label==` to find images without labels.